dist/images/Dockerfile

```diff
@@ -21,17 +21,14 @@ USER root
 ENV PYTHONDONTWRITEBYTECODE yes
 
 COPY kubernetes.repo /etc/yum.repos.d/kubernetes.repo
-RUN yum install -y  \
-	PyYAML bind-utils \
-	openssl \
-	numactl-libs \
-	firewalld-filesystem \
-	libpcap \
-	hostname \
-	kubectl \
+RUN INSTALL_PKGS=" \
```

Comment:
you don't need `INSTALL_PKGS`... that was needed with the `rpm -V ...` version of the patch because you needed to specify all the packages twice, but with the `--setopt=skip_missing_names_on_install=False` version you can just leave the packages inline like they were before